LEGO1/lego/sources/3dmanager/tglsurface.cpp

```diff
@@ -84,7 +84,7 @@ BOOL TglSurface::Create(const CreateStruct& rCreateStruct, Renderer* pRenderer,
 	BOOL dither = TRUE;
 	int textureShadeCount = -1;
 	int textureColorCount = -1;
-	Result result;
+	Tgl::Result result;
```

Comment:
What exactly is the type conflict?

Reply:
libctru (and therefore a handful of 3DS code) has a definition for `Result`: https://libctru.devkitpro.org/types_8h_source.html

This makes ambiguity for the compiler, so probably in another PR, some sort of "disambiguation" will need to be done, doing this or renaming Result to TglResult, unless there's some other solution I don't know of.